File: Hello_World.py
'''
# Hello World program with execution time measurement'''
import time
# Start timing
start_time = time.time()
# Print Hello, World!
print("Hello, World!")
# End timing and calculate execution time
end_time = time.time()
exec_time = (end_time - start_time) * 1000  # milliseconds
print(f"Execution time: {exec_time} ms")

File: Hello_World.cpp
#include <iostream>
#include <chrono>
/*
A simple C++ program that prints "Hello, World!" and measures execution time.
*/
int main() {
    // Start timing
    auto start = std::chrono::high_resolution_clock::now();

    // Print Hello, World!
    std::cout << "Hello, World!" << std::endl;

    // End timing and calculate execution time
    auto end = std::chrono::high_resolution_clock::now();
    std::chrono::duration<double, std::milli> exec_time = end - start;
    std::cout << "Execution time: " << exec_time.count() << " ms" << std::endl;

    return 0;
}
